config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get "password_resets/new"
+  get "password_resets/edit"
```

Comment:
Do you still need routing in the line 3 and 4? If so, them why you need those routing?